schedpolicy/sched_policy.go

```diff
@@ -0,0 +1,52 @@
+//go:generate mockgen -package=mock -destination=mock/schedpolicy.mock.go github.com/libopenstorage/openstorage/schedpolicy SchedulePolicy
+package schedpolicy
+
+import "errors"
+
+var (
+	ErrNotImplemented = errors.New("Not Implemented")
+)
+
+type SchedulePolicy interface {
+	// SchedPolicyCreate creates a policy with given name and schedule.
+	SchedPolicyCreate(name, sched string) error
+	// SchedPolicyUpdate updates a policy with given name and schedule.
+	SchedPolicyUpdate(name, sched string) error
+	// SchedPolicyDelete deletes a policy with given name.
+	SchedPolicyDelete(name string) error
+	// SchedPolicyEnumerate enumerates all configured policies or the ones specified.
+	SchedPolicyEnumerate([]string) ([]*SchedPolicy, error)
+}
+
+type Manager struct {
+	SchedulePolicy
+}
+
+func NewSchedulePolicyManager(sched SchedulePolicy) *Manager {
+	return &Manager{
+		SchedulePolicy: sched,
+	}
+}
+
+type nullSchedMgr struct {
+}
+
+func New() *nullSchedMgr {
```

Comment:
This one (and the Secret one) should probably have their name name from `New` to `NewNullSchedManager` because if not, when reading the server.go:120 looks like a real manager is being allocated.